vector/src/main/java/im/vector/app/features/settings/VectorSettingsPreferencesFragment.kt

```diff
@@ -103,6 +103,7 @@ class VectorSettingsPreferencesFragment :
 
         findPreference<Preference>(VectorPreferences.SETTINGS_PREF_SPACE_CATEGORY)!!.let { pref ->
             pref.isVisible = !vectorFeatures.isNewAppLayoutFeatureEnabled()
+            pref.isEnabled = !vectorPreferences.isNewAppLayoutEnabled()
```

Comment:
This has to be refreshed as well (like in `configureUnreadNotificationsAsTabPreference()`). Maybe rename `configureUnreadNotificationsAsTabPreference()` and use it to refresh all pref enable state?

Reply:
sorry, please ignore, this is not the same Fragment.